resources/public/index.html

```diff
@@ -3,24 +3,15 @@
   <head>
     <title>Hello, World!</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/main.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/styles/darkula.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.8.0/highlight.min.js"></script>
```

Comment:
+1. I'm not sure how this is commonly done with clojurescript. There's probably a lein plugin that integrates with bower or something.